### DISCUSSION TITLE
Enhancement:

### DISCUSSION THREAD

**Discussion by TheDiamonddd** [FunkinCrew/Funkin#1580] [Ideas]
I can't see the arrows because of my hands
I suggest move the buttons to another position

Example
⬅️⬆️⬇️➡️

.⬅️.................➡️
......⬆️......⬇️

Or

...⬅️..⬆️..⬇️.➡️

**Comment by regulad** [comment]
What?

**Comment by SilverT4** [comment]
it's possible to use the WASD keys as well if you're having trouble with the arrow keys.
